src/__tests__/Popover.spec.js

```diff
@@ -513,5 +513,17 @@ describe('Popover', () => {
 
       wrapper.detach();
     });
+
+    it('should pass down offset', ()=> {
+      const wrapper = mount(
+        <Popover
+          isOpen
+          target="innerTarget"
+          offset='100'>Popover content</Popover>
```

Comment:
- Unexpected usage of singlequote , use " " instead
- The closing bracket must be aligned with the line containing the opening tag